src/mame/drivers/sfbonus.c

```diff
@@ -462,7 +462,7 @@ class sfbonus_state : public driver_device
 	TILE_GET_INFO_MEMBER(get_sfbonus_reel4_tile_info);
 	virtual void machine_reset();
 	virtual void video_start();
-	void draw_reel_layer(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect, int catagory);
```

Comment:
Please don't mess with variable names if you aren't doing any real work on related code - it will screw up peoples autocomplete indexing, ctags, etc.  I know it can be irritating to see, but it actually harms people's ability to get things done to mess with code like this.